dev/cnf/oss_dependencies.maven

```diff
@@ -545,9 +556,11 @@ org.shredzone.acme4j:acme4j-client:2.7
 org.shredzone.acme4j:acme4j-utils:2.7
 org.slf4j:slf4j-api:1.7.7
 org.slf4j:slf4j-api:1.7.26
+org.slf4j:slf4j-api:1.7.28
 org.slf4j:slf4j-jdk14:1.7.7
 org.slf4j:slf4j-simple:1.7.25
 org.slf4j:slf4j-simple:1.7.26
+org.slf4j:slf4j-simple:1.7.28
```

Comment:
We don't need all these different versions of third party libraries! Please just use the latest one that is already there ... and if I were doing this, I would attempt to remove the older versions. Anything that uses a version of SLF4J 1.7 should work just fine with the latest bug fix version; it might even work better!

Reply:
Will create a separate issue to consolidate the slf4j-simple versions.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/13184